tests/codegen/intrinsics/transmute.rs

```diff
@@ -190,7 +190,7 @@ pub unsafe fn check_byte_from_bool(x: bool) -> u8 {
 // CHECK-LABEL: @check_to_pair(
 #[no_mangle]
 pub unsafe fn check_to_pair(x: u64) -> Option<i32> {
```

Comment:
That test should be part of https://github.com/rust-lang/rust/pull/116672, as we don't get different layout for i128 with just this PR (we only get it for bool -- for everything else the difference would be in getelementptr indices rather than load/store layout).